Docs/developer_guide/build_instructions/windows.md

```diff
@@ -56,20 +56,28 @@ Specify source, build, and Qt location and compiler version and start the build
 
 Release mode:
 
-```
-mkdir C:\D\S4R
-cd /d C:\D\S4R
-"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 17 2022" -A x64 -DQt5_DIR:PATH=C:\Qt\5.15.2\msvc2019_64\lib\cmake\Qt5 C:\D\S4
-"C:\Program Files\CMake\bin\cmake.exe" --build . --config Release
+```bat
+cd C:\D
+
+"C:\Program Files\CMake\bin\cmake.exe" ^
+  -G "Visual Studio 17 2022" -A x64 ^
+  -DQt5_DIR:PATH=C:/Qt/5.15.2/msvc2019_64/lib/cmake/Qt5 ^
+  -S C:\D\S4 -B C:\D\S4R
```

Comment:
“S4” was short for Slicer version 4, however we are now on Slicer 5 versions so “S5” would be more accurate as a short description.

The change to S4 to S5 would be propagated elsewhere as well.
```suggestion
  -S C:\D\S5 -B C:\D\S5R
```

Reply:
Could we change it to S instead of S5 so it doesn't need to be changed for each new major version?

Reply:
That makes sense.

Reply:
I have gone ahead and pushed a commit with that change.